analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/DisablingCsrfProtectionTest.cs

```diff
@@ -28,26 +28,28 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class DisablingCsrfProtectionTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder().WithBasePath("Hotspots").AddAnalyzer(() => new DisablingCsrfProtection(AnalyzerConfiguration.AlwaysEnabled));
+
         [TestMethod]
         public void DisablingCSRFProtection_CS() =>
```

Comment:
I would name this `CSharp9` instead of `CS` to be consistent with the other test names.